infra/aws/terraform/registry.k8s.io/s3/bucket.tf

```diff
@@ -15,7 +15,8 @@ limitations under the License.
 */
 
 resource "aws_s3_bucket" "registry-k8s-io" {
-  bucket = "${var.prefix}registry-k8s-io-${var.region}"
+  bucket = "${var.prefix}registry-k8s-io-${data.aws_region.current.name}"
+  force_destroy = true
```

Comment:
@deobieta, is there any particular reason why `force_destroy` is set to to true here?

Reply:
Oh, for testing purposes. I've deleted that line. Thanks.